<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(172, 125, 136);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(248, 236, 209); 
font: 700 italic 13pt &quot;Garamond&quot;</string>
   </property>
   <property name="title">
    <string>  Personal Information  </string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
    </property>
    <property name="text">
     <string>Student ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
    </property>
    <property name="text">
     <string>Department:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
    </property>
    <property name="text">
     <string>Year Level:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbDept">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbCourse">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbLevel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>SEARCH RECORDS:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboRecord">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>EDIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>FILL IN THE BOXES IN CAPITAL LETTERS. THANKYOU!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
